auditai/misc.py

```diff
@@ -401,3 +401,64 @@ def test_multiple(labels, decisions,
             print("%s: %f" % (key, results[key]))
 
     return results
+
+
+def quick_bias_check(clf, df, 
```

Comment:
Can we change variable name of the df argument to make it more informative - something like reference_df or test_df would be better.

Reply:
i chose the variable names to be similar to existing functions, e.g. here: https://github.com/pymetrics/audit-ai/blob/cbc202257836b3140de87903ada1e3376b802bc5/auditai/misc.py#L112

if you want to rename them all to be more informative, i'd be ok with that but we shouldn't be inconsistent

Reply:
That's fair. Ideally we'd like the function arguments to have informative variable names, but we can keep it to df so that it's consistent.